dask/array/reductions.py

```diff
@@ -440,6 +442,12 @@ def vnorm(a, ord=None, axis=None, dtype=None, keepdims=False, split_every=None,
 
     See np.linalg.norm
     """
+
+    warnings.warn(
+        "Please use `dask.array.linalg.norm` instead.",
+        DeprecationWarning
+    )
```

Comment:
Deprecation warnings don't show up by default in python 3. We've been using `warnings.warn` with the default `UserWarning`, and prefixing the string with `"DeprecationWarning:"`. See https://github.com/dask/dask/blob/master/dask/dataframe/rolling.py#L150-L152 for an example.

Reply:
Interesting. Thanks for the tip. What was their rationale for this change?

Reply:
Not sure. Changed in python 3.2.